### ISSUE TITLE
prisma error {"clientVersion":"3.8.1"} in app engine

### ISSUE THREAD

**Issue by worktechnotoil** [prisma/prisma#11533]
### Bug description

this is my prisma code .when i try to deploy prisma + nodejs into app engine . i got this error message .everything working fine in locally . but when i try to deploy the code in app engine then i got this error message .

 {"clientVersion":"3.8.1"} 

```
const express = require("express");
const route = express();
const { PrismaClient } = require("../prisma/generated/client");
const prisma = new PrismaClient();

route.get("/", async (req, res) => {
  try {
    const allUsers = await prisma.account.findMany();

    res.send({
      result: allUsers,
    });
  } catch (error) {
    res.send({
      error: error,
    });
  }
});

module.exports = route;
```

### Prisma Version

```
prisma                  : 3.8.1
@prisma/client          : 3.8.1
Current platform        : darwin
Query Engine (Node-API) : libquery-engine 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/libquery_engine-darwin.dylib.node)
Migration Engine        : migration-engine-cli 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine    : introspection-core 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary           : prisma-fmt 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/prisma-fmt-darwin)
Default Engines Hash    : 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f
Studio                  : 0.452.0
```

**Comment by Jolg42** [comment]
Hi @worktechnotoil 

Could you provide a reproduction repository, maybe?
The error is a bit weird, could you add a screenshot?

What did you try? What didn't work? (If you could let us know which steps you did so we can get to a reproduction)

Does your example here worked locally but not when deployed, maybe?

**Comment by worktechnotoil** [comment]
> Hi @worktechnotoil
> 
> Could you provide a reproduction repository, maybe? The error is a bit weird, could you add a screenshot?
> 
> What did you try? What didn't work? (If you could let us know which steps you did so we can get to a reproduction)
> 
> Does your example here worked locally but not when deployed, maybe?

**Comment by worktechnotoil** [comment]
@Jolg42  , yep exactly worked in locally but not worked in deployment .this is the actual response i got in try catch block .
{"clientVersion":"3.8.1"} thats it .

**Comment by worktechnotoil** [comment]
prisma have any docs for google app engine deployment ?

**Comment by worktechnotoil** [comment]
[11385](https://github.com/prisma/prisma/issues/11385) check this issue also .same issue .

**Comment by Jolg42** [comment]
@worktechnotoil I see you closed the issue, maybe by mistake?

**Comment by worktechnotoil** [comment]
> @worktechnotoil I see you closed the issue, maybe by mistake?

i think yes .

**Comment by worktechnotoil** [comment]
should i create the new issue again ?

**Comment by sniper365** [comment]
I am using prisma -v 4.5.0.

I was following the guide [](https://planetscale.com/docs/tutorials/connect-nextjs-app)

At first it worked on local and on vercel. But after refactoring I faced this error..
I am very struggling to fix this. But couldn't find the solution yet.

Is there any relation about previous comments? Could you anyone help me?

Thank you

**Comment by Jolg42** [comment]
Probably related to https://github.com/prisma/prisma/issues/11385, could you [open a new issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) and add a link to it?

**Comment by millsp** [comment]
Reproduction https://github.com/prisma/prisma/pull/16231

**Comment by ItaloEC** [comment]
same problem here..
in local, it works perfectly but in vercel, it returns `"clientVersion": "4.6.1"` in the catch block

**Comment by ABSanthosh** [comment]
@ItaloEC I had the same issue now. I had ` "@prisma/client": "^4.1.1"` and I updated it to `"@prisma/client": "^4.7.0"` and its fixed.

**Comment by darkodavidovic** [comment]
Solution:
1: npm uninstall prisma @prisma/client
2: npm install -D prisma
3: npm install @prisma/client
4: npx prisma generate
Note: Try/Catch block will return {"clientVersion":"x.x.x"} as error even if your data inside create/update has wrong value pairs in your payload you want to create/update.
Note2: Error can occur when you use prisma with SQLITE. In MySQL/Postgres, you can see what is the error more accurate.
Note3: Some prisma features are not available with SQLITE, such as error reporting/createMany metod, etc...

**Comment by ABSanthosh** [comment]
> Note: Try/Catch block will return {"clientVersion":"x.x.x"} as error even if your data inside create/update has wrong value pairs in your payload you want to create/update.

Yes! this was the issue. Remove try-catch and call the function to debug.

**Comment by JanPoonthong** [comment]
Did you add `0.0.0.0/0` in MongoDB? so that everyone can access it

**Comment by pablo-capicua** [comment]
Maybe this can help someone..

The same thing happened to me during a deployment. The issue was that the environment variables file (.env) was not being copied correctly, so Prisma was unable to connect to the database because the variable containing the database URL was essentially undefined. Any other API endpoint that did not use Prisma was working properly.

Additionally, while investigating the cause of the error, I also found that calling a method such as prisma.user.findUnique({ where: { email } }); would throw the same error if the email was undefined.
